napari/layers/shapes/_shape_list.py

```diff
@@ -1029,9 +1029,9 @@ def outline(
 
         Parameters
         ----------
-        indices : int | list
-            Location in list of the shapes to be outline. If list must be a
-            list of int
+        indices : int | Sequence[int]
+            Location in list of the shapes to be outline.
+            If sequence must be a list of int
```

Comment:
```suggestion
            If sequence, all elements should be ints
```

?

(ie it could be a tuple of int, yes?)